packages/suite-desktop/src-electron/electron.ts

```diff
@@ -159,6 +163,133 @@ const init = async () => {
     }
 
     mainWindow.loadURL(src);
+
+    // Check for updates when ready
+    ipcMain.on('ready', () => {
+        autoUpdater.checkForUpdates();
+
+        // And every hour
+        setInterval(() => autoUpdater.checkForUpdates(), 60 * 60 * 1000);
```

Comment:
Not sure, but is it safe not to unset `setInterval` later on? Probably it's safe but just double-checking.

Reply:
Some time ago we were discussing that we should think twice and be very carefuly when implementing any requests to our infrastructure from suite - especially in interval. If somethink goes very wrong, we could end up in state when we are ddos attacing ourselfs :D.

Reply:
That one is pointed at Github :sweat_smile: I think unsettling the interval is not necessary as it will get killed when the application is terminated. I think just to be on the safer side, I'll move the interval outside the 'ready' event (in case it gets called multiple times).